toml/region.hpp

```diff
@@ -92,7 +92,7 @@ struct location final : public region_base
     char front() const noexcept override {return *iter_;}
 
     // this const prohibits codes like `++(loc.iter())`.
-    const const_iterator iter()  const noexcept {return iter_;}
+    std::add_const_t<const_iterator> iter()  const noexcept {return iter_;}
```

Comment:
This fails in case of C++11. `foo_t` alias is not available until C++14.

```suggestion
    std::add_const<const_iterator>::type iter()  const noexcept {return iter_;}
```

Reply:
:) Looks like I've spent to much time in code bases with at lest C++17 available and forget that `_t` and `_v` were not always available.